src/protocols/aeternity/views/AccountDetails.vue

```diff
@@ -93,3 +100,12 @@ export default defineComponent({
   },
 });
 </script>
+
+<style lang="scss" scoped>
+@use '@/styles/variables';
+
+.account-ion-content {
+  overflow: hidden;
+  background-color: variables.$color-bg-4;
+}
```

Comment:
Why are these changes needed?

Reply:
The background is needed so that the animations do not look broken

Reply:
Don't we need it to be a more general thing or we need it only for this screen?

Reply:
There's also a more general class `.ion-content-bg` but with a different color and no `overflow: hidden;`
We could use the `.ion-content-bg--lighter` class which is the same color and only add `overflow: hidden;` with a new class for this screen